src/Client/BuzzHouse/Generator/FuzzConfig.h

```diff
@@ -75,45 +105,27 @@ static std::optional<ServerCredentials> loadServerCredentials(
     std::string database = "test";
     std::filesystem::path query_log_file = std::filesystem::temp_directory_path() / (sname + ".sql");
 
+    std::map<std::string, std::function<void(const JSONObjectType &)>> config_entries = {
```

Comment:
This is a change in the good direction to avoid the clutter with all the if statements. However, this can be made static. And most likely an `std::unordered_map`. You tend to use ordered maps a lot when I don't think we require them to be sorted.